src/licensedcode/data/licenses/ecfonts-1.0.yml

```diff
@@ -0,0 +1,8 @@
+key: latex-ec-fonts
```

Comment:
You did not rename the key here, therefore all your tests are failing. The key in the file name and in the key field must match

Reply:
@pombredanne Incorporated the changes in the files. Kindly review them. Sorry for that trivial mistake of not changing the key name :p Will take more care of these things in future commits. Thanks!